Simple Navbar + Wallet widget

desamtralized opened the pull request.

desamtralized committed:
wip: basic styling and navbar

MattNguyen commented on public/background.png:
can you remove this please? i really don't think we should bringing in the landing page styling. Branding is going to change.

MattNguyen commented on src/app/components/Navbar/index.tsx:
```diff
@@ -0,0 +1,61 @@
+
+    return (
+        <header className='fixed top-0 w-full z-40'>
+            <nav className={twJoin("w-full md:static md:text-sm flex", state && "fixed z-10 h-full", "md:bg-[#F5F1EB] bg-transparent")}>
```
as mentioned above, i dont think we should be bringing in landing page styling. the tailwinds overrides (eg `md:bg-[#F5F1EB`, `lg:w-[990px]`) forces the rest of the app to look a certain way when we should defer to the default sizing, scaling, color schemes, etc. If we want to create a theme, it should be done at the tailwinds config level. Otherwise we should leverage radix themes. Even better, we should use radix primitives with tailwinds classes so that there's some consistency.

desamtralized committed:
improve connect wallet widget styling

desamtralized committed:
Refactor ConnectModal and Wallet components

- Refactored `ConnectModal.tsx` to use `twMerge` for merging Tailwind classes, improving readability and maintainability.
- Adjusted `Wallet.tsx` to ensure consistent cursor pointer behavior on buttons and dropdown items.
- Simplified the JSX structure in `Wallet.tsx` for better clarity.

desamtralized committed:
remove unnecessary styling

desamtralized committed:
fix linting issues

desamtralized committed:
remove: unused libs, images & styles

desamtralized committed:
update lockfile
